features/google/src/views/import/import.jsx

```diff
@@ -31,7 +31,11 @@ const ImportView = () => {
         if (files?.[0]?.id) handleRemoveFile(files[0].id);
         runWithLoadingScreen(async function () {
             try {
-                await polyOut.importArchive(selectedFile.url);
+                const cleanedSelectedFileUrl = selectedFile.url
```

Comment:
This is essentially URI-decoding the url. If that's what you want to do, do it all the way using some URI-decoding function. The files could have other encoded features such as ß or ú.

Reply:
I will revert all changes as the problem lies somewhere else